src/main/kotlin/org/jitsi/nlj/rtcp/KeyframeRequester.kt

```diff
@@ -62,7 +62,14 @@ class KeyframeRequester : TransformerNode("Keyframe Requester") {
         } ?: return packetInfo
 
         val now = System.currentTimeMillis()
-        val canSend = canSendKeyframeRequest(pliOrFir.mediaSourceSsrc, now)
+        val sourceSsrc = when (pliOrFir) {
+            // PLIs use the generic "media source SSRC" field
+            is RtcpFbPacket -> pliOrFir.mediaSourceSsrc
```

Comment:
That's a bug, it should be RtcpFbPliPacket. But it's corrected in the next commit